README.md

```diff
@@ -4,42 +4,42 @@ wpantund, Userspace WPAN Network Daemon
 `wpantund` is a user-space network interface driver/daemon that
 provides a native IPv6 network interface to a low-power wireless
 **Network Co-Processor** (or *NCP*). It was written and developed by
-Nest Labs to make supporting [Thread](http://threadgroup.org)
-connectivity on unix-like operating systems more straightforward.
+Nest to support [Thread](http://threadgroup.org)
+connectivity on uUnix-like operating systems and make the support more straightforward.
 
-`wpantund` is designed to marshall all access to the NCP, ensuring
-that it always remains in a consistent, well-defined state.
+`wpantund` is designed to marshall all access to the NCP ensuring
```

Comment:
I'll admit that I **do** have a problem with unnecessary commas, but that first comma seems important to maintain the logical flow between the two separate thoughts of this sentence:

* `wpantund` is designed to marshall all access to the NCP.
* `wpantund` ensures the NCP remains in a consistent and well-defined state.

Removing it entirely seems to make the ideas run together awkwardly. Replacing it with a semicolon is too heavy-handed.

Thoughts?